### PR TITLE
Play area bounds

### PR DESCRIPTION
**Pull request by EthanNichols**
# Description

Player movement is based off of Rigidbody velocity instead of manually updating the position.
Bounds are generated by the PlayArea script. Which matches to the screen view.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## How were the changes tested

Tested in my scene.

Checked to see if the bounds were generated along the camera viewport.
Made sure the player couldn't leave the view of the camera.
Made sure the player could still move.

## Self-Review Checklist:

- [X] Code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings or errors

### CODE REVIEW

**Comment by EthanNichols** [comment]
Going to wait for the "Player Firing" PR to get merged before this gets merged. Since there are changes to the PlayShip class.

**Review by LegendRK — APPROVED** [review]

**Pull request merged**
